### PR TITLE
After fixed

### PR DESCRIPTION
**Pull request by al4iii**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,103 +1,122 @@
...
-    MemoryNewNumber = false,
-    MemoryPendingOperation = "";
+let numbers = document.querySelectorAll('.number');
+let operations = document.querySelectorAll('.operator');
+let decimalBtn = document.getElementById('decimal');
+let clearBtns = document.querySelectorAll('.clear-btn');   
+let resultBtn = document.getElementById('result');
+let piBtn = document.getElementById('pi');
+let display =document.getElementById('display');
+let MemoryCurrentNumber = 0; // what number is currently entered on the scoreboard, by default 0
```
Comment (Yauhenia-Zhdanovich) [inline script.js:8]: name of variables should start with a lowercase letter

File: script.js [hunk 2]
```diff
@@ -1,103 +1,122 @@
...
-    MemoryCurrentNumber = 0,
-    MemoryNewNumber = false,
-    MemoryPendingOperation = "";
+let numbers = document.querySelectorAll('.number');
+let operations = document.querySelectorAll('.operator');
+let decimalBtn = document.getElementById('decimal');
+let clearBtns = document.querySelectorAll('.clear-btn');   
+let resultBtn = document.getElementById('result');
+let piBtn = document.getElementById('pi');
+let display =document.getElementById('display');
```
Comment (Yauhenia-Zhdanovich) [inline script.js:7]: spacing

File: script.js [hunk 3]
```diff
@@ -1,103 +1,122 @@
...
-        };
-        
-      }
+function numberPress(number){
+  if(MemoryNewNumber){
+    display.value = number;
+    MemoryNewNumber = false;
+  }else{
+  if(display.value === '0'){
+    display.value = number;
```
Comment (Yauhenia-Zhdanovich) [inline script.js:43]: use ternary operator for this logic

File: script.js [hunk 4]
```diff
@@ -1,103 +1,122 @@
...
+                MemoryCurrentNumber = display.value;               
+                MemoryCurrentNumber = 1/localOperationMemory;
+                MemoryPendingOperation = op;
+                break;
+            default:
+                MemoryCurrentNumber = +localOperationMemory; 
+                break;
+          };
+    display.value = MemoryCurrentNumber;
+    MemoryPendingOperation = op;
```
Comment (Yauhenia-Zhdanovich) [inline script.js:94]: is this necessary to have this piece  ` MemoryPendingOperation = op;` on 72, 77, 82, 87, and here?

File: script.js (hunk 1 above)
Comment (al4iii) [reply script.js:8]: done

File: script.js (hunk 4 above)
Comment (al4iii) [reply script.js:94]: done

File: script.js (hunk 3 above)
Comment (al4iii) [reply script.js:43]: done

**Pull request merged**